server/parsec/asgi/__init__.py

```diff
@@ -106,6 +93,12 @@ async def serve_parsec_asgi_app(
         ssl_keyfile=ssl_keyfile,  # type: ignore
         ssl_certfile=ssl_certfile,
         workers=workers,
+        # Enable/Disable X-Forwarded-Proto, X-Forwarded-For to populate remote address info.
+        # Currently uvicorn only supports X-Forwarded headers
+        # An issue exist to support Forwarded headers: https://github.com/encode/uvicorn/issues/2237
+        proxy_headers=(proxy_trusted_addresses != []),
+        # Comma separated list of IP Addresses, IP Networks, or literals (e.g. UNIX Socket path) to trust with proxy headers
+        forwarded_allow_ips=proxy_trusted_addresses,
```

Comment:
As stated in uvicorn docs regarding `forwarded_allow_ips`: 
> Defaults to the $FORWARDED_ALLOW_IPS environment variable if available, **or '127.0.0.1'**. 

Why change the default behavior? 
Why not leaving `proxy_headers` enabled (the default value) and setting `forwarded_allow_ips` as:

```suggestion
        forwarded_allow_ips=proxy_trusted_addresses+["localhost", "127.0.0.1", "::1"],
```

Reply:
The main reason is explicit vs implicit configuration.

Previously we cannot disable the parsing of forwarded header. Now we can and allow the user to provide a specific list.

I will not apply your suggestion as your are hard-coding values that cannot be overwritten by the user.

Reply:
OK, so you suggest to set those default values somewhere else?

Reply:
done, in another way (default value for option)